bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java

```diff
@@ -218,6 +224,8 @@ private void createComplete(int rc, LedgerHandle lh) {
         private byte[] builderPassword;
         private org.apache.bookkeeper.client.api.DigestType builderDigestType
             = org.apache.bookkeeper.client.api.DigestType.CRC32;
+        private org.apache.bookkeeper.client.api.LedgerType builderLedgerType
```

Comment:
why do you use the fully qualified class name here? don't you already import `LedgerType`?

Reply:
yup, it was a cut and paste from the DigestType which needs the FQN